src/MIDebugPackage/OpenFolderSchema.json

```diff
@@ -185,6 +185,15 @@
               },
               "hardwareBreakpoints": {
                 "$ref": "#/definitions/cpp_schema/definitions/hardwareBreakpointsOptions"
+              },
+              "unknownBreakpointHandling": {
+                "type": "string",
+                "enum": [
+                  "throw",
+                  "stop",
+                  "ignore"
+                ],
+                "description": "Controls how breakpoints set externally (usually via raw GDB commands) are handled when hit.\nAllowed values are \"throw\", which acts as if an exception was thrown by the application, \"stop\", which only pauses the debug session, and \"ignore\", which continues execution without pausing. The default value is \"throw\"."
```

Comment:
I would suggest putting in a warning for `ignore`. Maybe something like:

> Note that `"ignore"` can cause side effects such as causing step over to become a go, and poor performance. So, it should generally not be used.

Reply:
`ignore` is removed in latest update.